tests/debugger/test_debugger_probe_snapshot.py

```diff
@@ -144,7 +151,7 @@ def test_code_origin_entry_present(self):
             if resource == "GET /healthcheck" and resource_type == "web":
                 code_origin_type = span["meta"].get("_dd.code_origin.type", "")
                 code_origins_entry_found = code_origin_type == "entry"
-
+                
```

Comment:
Shouldn't the linter complain about this white-space?

Reply:
You've right. Interesting why CI didn't complain on it. I've removed the line here 3f3d169546265e65f9457f13b8348959e3b6ac8b